utils/e2e-tests/bash/tests/try-runtime/on_runtime_upgrade_live.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -euo pipefail
+
+HUMANODE_PEER_PATH="${1?Provide the path to the humanode peer as the first argument}"
+shift
+
+RUNTIME_WASM="${1?Provide the path to runtime wasm file as the second argument}"
```

Comment:
This might be better to pass via env vars...

Reply:
We can address this in a separate PR, as this would cause a refactor that's unrelated to this PR.